File: net/filtering-content/GetItemsOfTypes.cs
// DocSection: filtering_get_items_of_types
// Gets items based on the types Product, Article, and News
IDeliveryItemListingResponse<object> response = await deliveryClient.GetItemsAsync<object>(
    new InFilter("system.type", "product", "article", "news")
);
// EndDocSection

File: android/filtering-content/GetItemsOfTypes.java
// DocSection: filtering_get_items_of_types
// Gets items based on the types Article, Product, and News
List<ContentItem> items = deliveryService.<ContentItem>items()
    .inFilter("system.type", Arrays.asList("article", "product", "news"))
    .get()
    .getItems();
// EndDocSection

